guide/general/priority_list.md

```diff
@@ -99,6 +99,18 @@ Covenant | <input type="radio" id="ft-radio" name="cov" value="1" /><label for="
     - {{ site.data.spell.es }}
     - {{ site.data.spell.lb }}
 
+#### {{ site.data.talent.se }} Single Target cases
+
+In aoe you only cast {{ site.data.spell.cl }} and {{ site.data.spell.eq }} at 3+ targets. In ST you want to {{ site.data.spell.fs }} before casting {{ site.data.talent.se }} and with {{ site.data.spell.bloodlust }} then you only cast {{ site.data.spell.lb }} and {{ site.data.spell.es }} with using {{ site.data.spell.fs }} and use {{ site.data.spell.ls }} proc for movement. Without {{ site.data.spell.bloodlust }} then you cast {{ site.data.spell.ls }} procs, {{ site.data.spell.lb }} and {{ site.data.spell.es }} at the highest level of gear pre 9.1. At lower levels of gear normal rotation is advised outside of lust and using {{ site.data.spell.ls }} procs during lust.
+ 
+With {{ site.data.talent.eogs }} you still alternate between using {{ site.data.spell.es }} and {{ site.data.spell.eq }} to maintain the {{ site.data.talent.eogs }} buff.
+
+With {{ site.data.spell.primoridial wave }} you still cast it on cd. Favoring to use a {{ site.data.spell.ls }} but hardcasting {{ site.data.spell.lvb }} if needed to use the {{ site.data.spell.primoridial wave }} buff.
+
+**Tip**
+When using {{ site.data.lengendary.windspeaker }}, {{ site.data.talent.sk }} and {{ site.data.talent.mote }} on single target. You can go {{ site.data.spell.lava_burst }}, {{ site.data.spell.lb }} (empowered by {{ site.data.talent.sk }}), {{ site.data.spell.lava_burst }}, {{ site.data.spell.es }}, {{ site.data.spell.lava_burst }} (from {{ site.data.lengendary.windspeaker }}), {{ site.data.spell.lb }} (empowered by {{ site.data.talent.sk }})
```

Comment:
isnt that in the opener anyway?
if it isn't it should rather be there than in the main rotational section